src/less/components/button/index.less

```diff
@@ -1,6 +1,9 @@
-/* ================================================== *\
- * MIXINS
-\* ================================================== */
+// ================================================== //
+// INTERNALLY DEPRECATED! Use SCSS, instead!          //
+// ================================================== //
+
```

Comment:
Styles are still present in this file because other Light DOM styles in both `src/` and `docs/` reference this file. This file will eventually be removed with further LESS-to-SASS conversion in the future.